pkg/cmd/roachtest/scrub.go

```diff
@@ -57,6 +57,9 @@ func makeScrubTPCCTest(
 				Warehouses: warehouses,
 				Extra:      "--wait=false --tolerate-errors",
 				During: func(ctx context.Context) error {
+					// Wait until tpcc has been running for a few minutes to start SCRUB checks
+					time.Sleep(time.Minute * 5)
```

Comment:
probably better to use length/10

Reply:
I picked 5 minutes just because it's the same as the ramp up time (https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/tpcc.go#L49).

Reply:
I suspect this is also going to apply for when you run the roachtest locally using --local. You'll have to disable this sleep when you run it locally by checking for the flag. I'd also make the sleep interval 2 * rampDuration and ensure it is less than length/2.

Thanks

Reply:
Done.

I also increased `numScrubRuns` in the most recent revision based on timing from the last nightly test run.